python_modules/dagster/dagster/_core/storage/legacy_storage.py

```diff
@@ -607,16 +608,14 @@ def free_concurrency_slots_for_run(self, run_id: str) -> None:
     def free_concurrency_slot_for_step(self, run_id: str, step_key: str) -> None:
         return self._storage.event_log_storage.free_concurrency_slot_for_step(run_id, step_key)
 
-    def get_asset_check_executions(
+    def get_asset_check_execution_history(
         self,
-        asset_key: AssetKey,
-        check_name: str,
+        key: "AssetCheckKey",
```

Comment:
`key` is very ambiguous. We cannot expect the method name to hint at what sort of key we want here.

```suggestion
        asset_check_key: "AssetCheckKey",
```